octoprint_mrbeam/static/js/messages.js

```diff
@@ -241,7 +241,11 @@ $(function () {
                 var lmid = -1;
                 self.messages().forEach(function (myMessage) {
                     lmid = Math.max(lmid, myMessage.id);
-                    if (myMessage.notification && myMessage.id > self.lastMessageId) {
+                    if (myMessage.notification && (myMessage.id > self.lastMessageId
+                    // The below condition is only added to keep the 0.11.0 2-step update message notification showing
+                    // if the user stayed on the 0.11.0rc1 version and didn't execute the 2nd software update
+                    // TODO: Should be removed after 0.11.0 stable release
+                    || (myMessage.id === 3 && MRBEAM_PLUGIN_VERSION === "0.11.0rc1"))) {
```

Comment:
will the message.id always be 3 for the update message? 🤔

Reply:
Yes (but only as solid as it can be) 
Added to the comment